pandas/_libs/groupby_helper.pxi.in

```diff
@@ -14,26 +14,22 @@ _int64_max = np.iinfo(np.int64).max
 
 {{py:
 
-# name, c_type, dest_type, dest_dtype
-dtypes = [('float64', 'float64_t', 'float64_t', 'np.float64'),
-          ('float32', 'float32_t', 'float32_t', 'np.float32')]
+# name, c_type
```

Comment:
is c_type the name we are calling this in other files? ideally try to be consistent

Reply:
There's some variation, but c_type is a common one.